src/opm/io/eclipse/ERst.cpp

```diff
@@ -160,7 +183,19 @@ void ERst::initSeparate(const int number)
     this->reportLoaded[number] = false;
 }
 
-int ERst::getArrayIndex(const std::string& name, int number) const
+std::tuple<int,int> ERst::getIndexRange(int reportStepNumber) const {
+
+    if (!hasReportStepNumber(reportStepNumber)) {
+        std::string message = "Trying to get index range for non existing sequence " + std::to_string(reportStepNumber);
+        OPM_THROW(std::invalid_argument, message);
+    }
+    
+    auto range_it = arrIndexRange.find(reportStepNumber);
```

Comment:
No bug deal - I'll merge this now; but you do seem to sprinkle your code with blank lines somewhat randomly?